.github/workflows/commit.yml

```diff
@@ -49,13 +49,10 @@ jobs:
       - name: Publish the Docker images
```

Comment:
Suggest moving this after the README publishing, so if the README cannot be updated the docker image is not updated